llvm/tools/bugpoint-passes/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ if( NOT LLVM_REQUIRES_RTTI )
 endif()
 
 if(WIN32 OR CYGWIN)
-  set(LLVM_LINK_COMPONENTS Core)
+  set(LLVM_LINK_COMPONENTS Core Support)
```

Comment:
I believe this should be committed to llvm.org since this is not SYCL specific.

Reply:
Hm.. okay, I'll make a PR for them as well.

Reply:
Unfortunately, LLVM project doesn't accept contributions via GitHub pull request. They use Phabricator.

From https://llvm.org/docs/Contributing.html#how-to-submit-a-patch:

> We don’t currently accept GitHub pull requests, and you’ll need to send patches via [Phabricator#phabricator-reviews](https://llvm.org/docs/Phabricator.html#phabricator-reviews).

Let me know if you have any troubles with submitting patches to LLVM.

Reply:
Okay, I'll have to remember how to use Phabricator, we used it for almost five years here in KDE :)

Reply:
Done: https://reviews.llvm.org/D140609

Reply:
Thanks! I suggest we add Chandler Carruth to reviewers. He is the code owner for CMake files and should be able to help with reviewing this patch.

Reply:
Added!